src/Avatar/Avatar.js

```diff
@@ -88,7 +84,19 @@ export type Props = {
   srcSet?: string,
 };
 
-function Avatar(props: DefaultProps & Props) {
+export type Props = CommonProps & {
```

Comment:
It's too bad flow isn't smart enough and require to export the Props to override our component.